### ISSUE TITLE
no spkit.cwt

### ISSUE THREAD

**Issue by TomasSand** [Nikeshbajaj/spkit#5]
Dear Nikeshbajaj, your toolkit is beautiful, but i have a little problem with spkit.cwt.
When import this, then an error: No module named 'spkit.cwt'

But the installation process it's ok.

thanks for any response
Tomas

**Comment by Nikeshbajaj** [comment]
Hi Tomas, 

Try either

`from spkit import cwt`

Or import those functions, you want to use from cwt as:

```
from spkit.cwt import ScalogramCWT
from spkit.cwt import compare_cwt_example
```
As it is in the example shown in documentation  - [https://spkit.github.io/docs/docs/cwt.html](https://spkit.github.io/docs/docs/cwt.html)


Or - third way is to import `spkit` as use functions with` spkit.cwt.function_name`

```
import spkit as sp
sp.cwt.compare_cwt_example(x,t,fs=fs)
XW,S = sp.cwt.ScalogramCWT(x,t,fs=fs,wType='Gauss',PlotPSD=True)
```

**Comment by Nikeshbajaj** [comment]
I hope that fixes the issue #5
